package.json

```diff
@@ -29,16 +41,18 @@
   "scripts": {
     "start": "tsdx watch",
     "clean": "rm -rf dist",
-    "build:cjs": "./node_modules/.bin/tsc -p tsconfig.cjs.json",
+    "build:cjs": "yarn tsc -p tsconfig.cjs.json",
     "build:umd": "webpack",
-    "build": "yarn clean && yarn build:cjs && yarn build:umd",
+    "build:esm": "yarn tsc -p tsconfig.esm.json",
+    "build": "yarn clean && yarn build:cjs && yarn build:umd && yarn build:esm",
     "test": "tsdx test ./test",
     "lint": "tsdx lint src test",
-    "prepare": "yarn lint && yarn build && yarn test",
+    "prepare": "yarn lint && yarn build",
     "publish:beta": "yarn publish --tag=beta"
   },
   "dependencies": {
-    "@pedrouid/iso-random": "^1.1.0",
+    "@pedrouid/environment": "^1.0.1",
+    "@pedrouid/iso-random": "^1.2.0",
```

Comment:
You'll want a version bump for this file too. And I wonder if a major version bump would be good for this refactor?